src/rpdk/core/build_image.py

```diff
@@ -0,0 +1,59 @@
+"""This sub command will build an image for the executable
+
+Projects can be created via the 'init' sub command.
```

Comment:
nit: This can be removed.  We have this comment for almost all the commands, but it's not useful.

Reply:
Removed